apps/frontend/src/firebase.js

```diff
@@ -2,15 +2,15 @@ import { initializeApp } from 'firebase/app';
 import { getStorage } from 'firebase/storage';
 
 const firebaseConfig = {
```

Comment:
Create "config.js" file, and export an object. Then import to this file

{
 firebase: ...
}